Tests/aweXpect.Tests/Numbers/ThatNumber.IsNotFinite.Tests.cs

```diff
@@ -133,7 +176,7 @@ await That(Act).Throws<XunitException>()
 					.WithMessage($"""
 					              Expected that subject
 					              is not finite,
-					              but it was {Formatter.Format(subject)}
+					              but it was {ValueFormatters.Format(Formatter, subject)}
```

Comment:
This change occurs multiple times and should be reverted

Reply:
Fixed in [f9c9d59](https://github.com/aweXpect/aweXpect/pull/486/commits/f9c9d59deb871d9064bb393b711c3ee3ad05f972)